pages/buildkite_compute/overview.md

```diff
@@ -0,0 +1,82 @@
+# What is compute
+
+Buildkite compute provides a fully managed platform for you to run your agents.
+
+With Compute Services, the infrastructure management tasks traditionally handled by your team, such as provisioning, scaling, and maintaining the servers that run your agents, can now be managed by Buildkite.
+
+Buildkite compute is currently in private trials, you need to contact support to express interest and have the service switched on for your organization.
```

Comment:
How is this interest expressed to support? Would customers just request support of this feature or is there an initial vetting process required between customers expressing an interest and this feature being activated for them?

```suggestion
Buildkite compute is currently in private trials. You'll need to contact support to express interest and have this service activated for your organization.
```